### PR TITLE
{bio}[intel/2020a] Metagenome-Atlas v2.4.3 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

requires easyblock:
- ~~https://github.com/easybuilders/easybuild-easyblocks/pull/2219~~ (~~no longer needed~~, now using `PythonBundle`)
  - needed for instalation of Metagenome-Atlas itself as extension as this includes several customized files.

requires easyconfigs:
- ~~#11617~~ (BBMap)
- ~~#11618~~ (CheckM)
- ~~#11619~~ (snakemake)

### CODE REVIEW

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/Metagenome-Atlas/Metagenome-Atlas-2.4.3-intel-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+# easyblock has to be named differently
+# otherwise eb tries to use it in extension installation as well, which fails.
+easyblock = 'EB_Metagenome_minus_Atlas_Bundle'
```
[the remark this answers is not in the document]
Comment (Darkless012) [reply easybuild/easyconfigs/m/Metagenome-Atlas/Metagenome-Atlas-2.4.3-intel-2020a-Python-3.8.2.eb:6]: Revisiting it now, yes I should and will. Didn't realize that extension could have their custom EasyBlock.

**Review by boegel — CHANGES_REQUESTED** [review]
@Darkless012 Please include `sanity_pip_check` in the easyconfig, which will raise a couple of missing extensions:

```
== 2020-11-24 09:20:17,121 easyblock.py:3335 WARNING build failed (first 300 chars): cmd "pip check" exited with exit code 1 and output:
ruamel-yaml 0.16.12 requires ruamel-yaml-clib, which is not installed.
plotly 4.11.0 requires retrying, which is not installed.
cookiecutter 1.7.2 has requirement poyo>=0.5.0, but you have poyo 0.2.0.
```

Also include a comment + mention author on top of the patch.

And close https://github.com/easybuilders/easybuild-easyblocks/pull/2219, since it's no longer needed now `PythonBundle` is used?

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 2219"

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2219
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3145.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/59094af491305322492fd6d519d8fb37 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2219
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2728.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/0428e0b40f479740e6ac4724757bedca for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**